src/pages/Staking/index.tsx

```diff
@@ -278,6 +280,15 @@ const Staking = (): JSX.Element => {
     }
   );
 
+  React.useEffect(() => {
+    (async () => {
+      if (bridgeLoaded && !!address) {
+        const newTransactionFeeReserve = await getStakingTransactionFeeReserve(address);
+        setTransactionFeeReserve(newTransactionFeeReserve);
+      }
+    })();
+  }, [bridgeLoaded, address]);
```

Comment:
- try/catch block might be necessary to catch errors.
- We can use react-query as it's more declarative and it exposes all loading states.